1-js/02-first-steps/15-function-basics/2-rewrite-function-question-or/task.md

```diff
@@ -2,25 +2,25 @@ importance: 4
 
 ---
 
-# Rewrite the function using '?' or '||'
+# '?' या '||' का उपयोग करके function को फिर से लिखें|
 
-The following function returns `true` if the parameter `age` is greater than `18`.
+यदि `age` parameter `18` से अधिक है, तो निम्नलिखित function `true` लौटाता है।
 
-Otherwise it asks for a confirmation and returns its result.
+अन्यथा, यह एक पुष्टिकरण मांगता है और अपना परिणाम देता है:
 
 ```js
 function checkAge(age) {
   if (age > 18) {
     return true;
   } else {
-    return confirm('Did parents allow you?');
+    return confirm("क्या माता-पिता ने आपको अनुमति दी थी?");
   }
 }
 ```
 
-Rewrite it, to perform the same, but without `if`, in a single line.
+इसे फिर से लिखें, वही प्रदर्शन करने के लिए, लेकिन बिना `if` के, एक पंक्ति में।
```

Comment:
Can we use something more explicit than `प्रदर्शन ` ? The translation doesn't need to match word by word if it's restricting us to convey the info.